SABnzbd.py

```diff
@@ -1283,7 +1283,7 @@ def main():
         cpumodel = getcpu()  # Linux only
         if cpumodel:
             logging.debug('CPU model name is %s', cpumodel)
-
+            
```

Comment:
If I change it in my branch, and do Commit / Push / pull request, will it arrive correctly?

If not, I don't know how to correct this.